module/Olcs/view/layout/signin.phtml

```diff
@@ -1,12 +1,14 @@
 <?php
-    $this->doctype();
+    echo $this->doctype().PHP_EOL;
 
     // OLCS-26896 - make sure X-UA-Compatible value is not escaped on the internal login screen
     $this->headMeta()->setAutoEscape(false);
 ?>
 <html lang="en">
   <head>
     <?php echo $this->partial('partials/head', ['title' => $this->pageTitle()]); ?>
+    <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+    <script src="https://code.jquery.com/jquery-migrate-3.3.2.min.js"></script>
```

Comment:
We should be using the jQuery bundled with `olcs-static`.

Reply:
remnant from testing i missed. Removing.